build.rs

```diff
@@ -97,8 +97,10 @@ fn main() {
         println!("cargo:rustc-cfg=libc_thread_local");
     }
 
-    // Rust >= 1.62.0 allows to use `const_extern_fn` for "Rust" and "C".
-    if rustc_minor_ver >= 62 {
+    // Rust > 1.62.0 allows to use `const_extern_fn` for "Rust" and "C".
```

Comment:
This isn't true for stable/beta and doesn't have to change, I think.

```suggestion
    // Rust >= 1.62.0 allows to use `const_extern_fn` for "Rust" and "C".
```